cmd/relayer/main.go

```diff
@@ -158,15 +189,29 @@ func run(c *cli.Context) error {
 		EthClient:             ec,
 		Forwarder:             contracts.NewIForwarderWrapped(forwarder),
 		Key:                   key,
-		ChainID:               chainID,
 		NewForwardRequestFunc: contracts.NewIForwarderForwardRequest,
+		ValidateTransactionFunc: func(_ *common.SubmitTransactionRequest) error {
+			// Note: an actual application will likely want to set this
+			return nil
+		},
```

Comment:
I'd rather not update the `NewForwardRequestFunc` here as it's unrelated to the PR.  but the function looks like this:
```
func NewIForwarderForwardRequest() common.ForwardRequest {
	return &IForwarderForwardRequest{}
}
```
are you suggesting making a type that implements this function, then passing an instance of that type in? 

I think the proper way to simplify `NewForwardRequestFunc` is to add it to the `Forwarder` interface since that's what it's tied to. I can do that in another PR